openapiart/openapiartgo.py

```diff
@@ -1402,7 +1402,7 @@ def _build_setters_getters(self, fluent_new):
                             fieldstruct=fieldstruct,
                             desc=self._api.components[fieldstruct].description.lstrip("// ")
                         )
-            if field.isOptional and field.isPointer:
+            if field.isOptional and field.isPointer or "StatusCode" in field.name:
```

Comment:
Also check fluent_new.isRpcResponse in combination with StatusCode since some fields may contain that string?